components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -28,6 +61,9 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "iam:policies";
     option (chef.automate.api.iam.policy).action = "iam:policies:create";
     };
+  /*
+  Get an IAM policy
```

Comment:
do these all need `IAM`? Seems a little redundant, unless we've already been referring to them like that in the docs.